confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConf.java

```diff
@@ -46,6 +49,15 @@ public class ConfigNodeConf {
   private TEndPoint[] configNodeGroupAddressList =
       Collections.singletonList(new TEndPoint("0.0.0.0", 22278)).toArray(new TEndPoint[0]);
 
+  /**
+   * ClientPool will have so many selector threads (TAsyncClientManager) to distribute to its
+   * clients.
+   */
+  private final int selectorNumOfClientPool =
+      Runtime.getRuntime().availableProcessors() / 4 > 0
+          ? Runtime.getRuntime().availableProcessors() / 4
+          : 1;
```

Comment:
Would it be better if make "selectorNumOfClientPool" configurable, and use Runtime.getRuntime().availableProcessors() / 4 as a default value?

Reply:
I've added it to the configuration